notifications.md

```diff
@@ -577,7 +601,7 @@ If you are sending an [on-demand notification](#on-demand-notifications), the `$
      * Get the mail representation of the notification.
      *
      * @param  mixed  $notifiable
-     * @return Mailable
+     * @return \Illuminate\Notifications\Messages\MailMessage
```

Comment:
This should still be a Mailable.
Because in this Section it is about returning a Mailable instead of a MailMessage.
```suggestion
     * @return Mailable
```